astroid/nodes/scoped_nodes/mixin.py

```diff
@@ -109,7 +109,7 @@ def _append_node(self, child: nodes.NodeNG) -> None:
         # which uses the current class as a mixin or base class.
         # It's rewritten in 2.0, so it makes no sense for now
         # to spend development time on it.
-        self.body.append(child)
+        self.body.append(child)  # type: ignore
```

Comment:
Some dynamic trickery happening here. From comment: `# pylint: disable=no-member; depending by the class`

Reply:
We should at least ignore a specific warning instead of the broader ignore.